.github/workflows/r.yml

```diff
@@ -143,6 +133,16 @@ jobs:
         with:
           fetch-depth: 0
           submodules: recursive
+      - name: Cache Docker Volumes
+        uses: actions/cache@88522ab9f39a2ea568f7027eddc7d8d8bc9d59c8 # v3.3.1
+        with:
+          path: .docker
+          # As this key is identical on both matrix builds only one will be able to successfully cache,
+          # this is fine as there are no differences in the build
+          key: ubuntu-${{ matrix.ubuntu }}-r-${{ matrix.r }}-${{ hashFiles('cpp/src/**/*.cc','cpp/src/**/*.h)') }}-${{ github.run_id }}
```

Comment:
the key doesn't work here as the values are not in the matrix, was this intentional?

Reply:
just saw the commit message^^ so this needs to be move up to it's intended place^^